### PR TITLE
Remove Polkadot & Kusama native runtime

### PR DESCRIPTION
**Pull request by bkchr**
This pull request removes the Polkadot and Kusama native runtime from the polkadot node. This brings some implications with it:

    There are no more kusama/polkadot-dev chain specs available. We will need to write some tooling in the fellowship repo to provide them easily.

    The try-runtime job for polkadot & kusama is not available anymore as we don't have the dev chain specs anymore.

    Certain benchmarking commands will also not work until we migrate them to use a runtime api.

Some crates in utils are still depending on the polkadot/kusama native runtime that will also need to be fixed.

Port of: https://github.com/paritytech/polkadot/pull/7467

### CODE REVIEW

**Comment by arkpar** [comment]
Why not remove westend and rococo as well?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3493435

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3503924

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3504039

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3504503

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3521290

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3612203

File: polkadot/utils/staking-miner/src/main.rs
Comment (ordian) [inline polkadot/utils/staking-miner/src/main.rs:1]: this was removed

**Review by ordian — APPROVED** [review]
Is this blocked on anything?

**Review by kianenigma — APPROVED** [review]

**Review by alvicsam — APPROVED** [review]

**Review by EgorPopelyaev — APPROVED** [review]

**Comment by bkchr** [comment]
> Is this blocked on anything?

Was blocked on reviews, which is now done :D

**Review by acatangiu — APPROVED** [review]

**Comment by liamaharon** [comment]
Is there a task tracking also removing SP runtimes from here?

**Comment by bkchr** [comment]
> Is there a task tracking also removing SP runtimes from here?

https://github.com/paritytech/polkadot-sdk/issues/603

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-release-analysis-v1-2-0/4451/1

**Pull request merged**